arbnode/node.go

```diff
@@ -467,6 +467,20 @@ func ConfigDefaultL1Test() *Config {
 	return &config
 }
 
+func ConfigDefaultL1NonSequencerTest() *Config {
+	config := ConfigDefault
+	config.Sequencer.Enable = false
+	config.L1Reader = headerreader.TestConfig
+	config.InboxReader = TestInboxReaderConfig
+	config.DelayedSequencer.Enable = false
+	config.BatchPoster.Enable = false
+	config.SeqCoordinator.Enable = false
+	config.Wasm.RootPath = validator.DefaultNitroMachineConfig.RootPath
+	config.BlockValidator = validator.TestBlockValidatorConfig
```

Comment:
Minor suggestion, but could config start as `ConfigDefaultL1Test()` to avoid duplicating the shared L1 config?

Reply:
done! (but the other way around)